ocw/lib/EC2.py

```diff
@@ -85,11 +86,14 @@ def all_regions(self):
         return regions
 
     def delete_instance(self, instance_id):
-        instances_list = list(self.ec2_resource().instances.filter(InstanceIds=[instance_id]))
-        if len(instances_list) > 0:
-            instances_list[0].terminate()
-        else:
-            logger.warning("Instance {} is ACTIVE in local DB but does not exists on EC2".format(instance_id))
+        try:
+            self.ec2_resource().instances.filter(InstanceIds=[instance_id]).terminate()
+        except ClientError as ex:
```

Comment:
```suggestion
        except ClientError as e:
```

Always seen `e` and never `ex`, but this is just aesthetics, nothing important.

Reply:
yeah I would agree that mostly `e` is used but I somehow prefer `ex` :)